docs/source/workflow_inputs.rst

```diff
@@ -1554,6 +1554,25 @@ Additional information is entered in each ``Ducts``.
          See the `Building America Solution Center <https://basc.pnnl.gov/resource-guides/ducts-buried-attic-insulation>`_ for more information.
   .. [#] DuctLocation choices are "conditioned space", "basement - conditioned", "basement - unconditioned", "crawlspace - unvented", "crawlspace - vented", "attic - unvented", "attic - vented", "garage", "outside", "exterior wall", "under slab", "roof deck", "other housing unit", "other heated space", "other multifamily buffer space", or "other non-freezing space".
          See :ref:`hpxmllocations` for descriptions.
+  .. [#] The sum of all FractionDuctArea must each equal to 1, both for the supply side and return side.
+  .. [#] FractionDuctArea or DuctSurfaceArea are required if DuctLocation is provided. If both are provided, DuctSurfaceArea will be used in the model.
```

Comment:
"FractionDuctArea or DuctSurfaceArea are required if DuctLocation is provided."

You can't simply copy what is in OS-HPXML, OS-ERI can be different. In this case, as shown in the table above, DuctLocation is required. So you should remove this and change the table to Required=Yes.

Reply:
Do we want to allow the user to provide both inputs?

Reply:
Probably we should. There may be software tools already writing both elements (e.g., if they get user inputs as fractions, convert to areas, and write both values to the HPXML) and we don't want to cause a breaking change for them.

Reply:
Got it. Then I guess we should keep the second note that reads "If both are provided, DuctSurfaceArea will be used in the model."